Taskfile.yaml

```diff
@@ -202,6 +202,7 @@ tasks:
       - tests/**/*
     cmds:
       - poetry run pytest --timeout=240 --junitxml=pytest.xml.0 -n auto --cov=featurebyte tests/unit
+      - poetry run pytest --timeout=240 --update-fixtures tests/unit/test_generate_payload_fixtures.py
```

Comment:
Would this regenerate the fixtures every time when running `task test-unit`? Is that the desired behaviour? I thought the `--update-fixtures` command is something that should be run manually only when needed.

Reply:
Yes. The main idea here is to run the actual fixture generation code (it will generate the fixtures but the changes will not be committed). Does it make sense to you?

Reply:
It seems a bit inconvenient for developers that the command results in many changed files that should not be committed when running locally. How about running without the `--update-fixtures` flag? Then, it would still run the fixture generation test but without writing out the fixtures. Would that catch the issue fixed by this PR?